app/assets/stylesheets/styles.scss

```diff
@@ -1,5 +1,98 @@
 @import url('https://fonts.googleapis.com/css?family=Open+Sans|Source+Sans+Pro');
+@font-face { 
+    font-family:"STARWARS";
+    src:url("star-wars-font/STARWARS.eot?") format("eot"),
+    url("star-wars-font/STARWARS.woff") format("woff"),
+    url("star-wars-font/STARWARS.ttf") format("truetype"),
+    url("star-wars-font/STARWARS.svg#STARWARS") format("svg");
+    font-weight:normal;
+    font-style:normal;
+}
```

Comment:
Basically most browsers support TTF, so we do not need all the other formats, they may just increase the size. It would be good to delete them from here and the files themselves also)